sbnanalysis/ana/CMakeLists.txt

```diff
@@ -18,9 +20,11 @@ link_directories(
   $ENV{NUSIMDATA_LIB}
   $ENV{LARCOREOBJ_LIB}
   $ENV{LARDATAOBJ_LIB}
+  $ENV{MRB_BUILDDIR}/uboonecode/lib/
```

Comment:
This'll be different depending on whether `uboonecode` is built in local products or a UPS version is used (and libs are in `$UBOONECODE_LIB`. Would it be possible to detect this, so it works either way?

Reply:
Fixed